metabase/metabase
# fix two y-axis each render splitlines

### Description

Fix two y-axis render their own splitlines which looks noisy. Previously, we rendered splitlines only for one axis.
Also closes https://github.com/metabase/metabase/issues/36824

### How to verify

- create a question with multiple series and assign them to different y-axes
- ensure the chart renders split lines only for left y-axis
- assign all series to the right axis
- ensure the chart renders split lines for right y-axis

### Demo

Labels on the screenshot are wrong but they will be fixed by https://github.com/metabase/metabase/pull/37721
<img width="718" alt="Screenshot 2024-01-16 at 9 04 17 PM" src="https://github.com/metabase/metabase/assets/14301985/1f93b7fd-045a-4697-a3bd-b703fde1fbd8">


<img width="992" alt="Screenshot 2024-01-16 at 9 05 58 PM" src="https://github.com/metabase/metabase/assets/14301985/cca9e5c8-0999-46f6-993c-2b79e56d0001">

### Checklist
